sample.env

```diff
@@ -0,0 +1,10 @@
+OPENAPI_KEY="seufhsfewnf-efhewbfbebifoewbfoe"
+DBURL="https://codemark-hbbdskswifeohefoewnwoilcnowncrgu.firebasedatabase.app/"
+FIREBASEAPIKEY="jfdkotaegiiuhKatiyarwljfehlfeis"
+AUTHDOMAIN= "codemark-eukwfgukpandeywbhdwbbdai.firebaseapp.com"
+PROJECTID="codemark-sdfgyiks6iru9vsrgni8dsu8"
+STORAGEBUCKET="codemark-ahkdikumarabiskb.appspot.com"
+MESSAGEINGSENDERID="duykfuyebcciuskehes"
```

Comment:
Also in the value, please use the dummy angle bracket value format, please?
FIREBASE_APP_ID = "\<Firebase APP ID from firebaseConfig\>"